src/malli/core.cljc

```diff
@@ -298,14 +298,15 @@
                :output output}
               max (assoc :max max)))))
 
+#?(:clj
+   (defn -entry-transformer [[k t]]
+     (fn [^Associative x]
+       (if-let [e ^MapEntry (.entryAt x k)]
```

Comment:
`.get` would be enough here?

Reply:
... with that, could the cljs code use the same code? Is there `Associative` in cljs?

Reply:
the transformers could be ordered by key, so the order would be consistent over cljs/cljs?

Reply:
> the transformers could be ordered by key, so the order would be consistent over cljs/cljs?

The problem with ordering is that composition reverses the order of application. Map keys have no order. Ideally, I would want a way to capture the scope of the current key, not only value, being transformed by the transformer

> `.get` would be enough here?

Don't think so. Your transformer could have a special way of handling `nil`s

> ... with that, could the cljs code use the same code? Is there `Associative` in cljs?

This requires someone with better knowledge of cljs than mine, which is 0.

Reply:
Even if we sort the transformers, to maintain consistency between the cljs and clj implementation we'll need to reverse the result in clj anyway.
Another concern, which is lesser but still exists, is the performance of compiling the schema elements. We already saw some pathological cases can make it worse. I wouldn't want to add more without good cause.